package.json

```diff
@@ -169,7 +169,6 @@
 		"@types/cookie": "^0.4.1",
 		"@types/debug": "^4.1.7",
 		"@types/fast-json-stable-stringify": "^2.0.0",
-		"@types/jest": "^25.2.3",
```

Comment:
Was this dependency unneeded, or is this change caused by the migration to Emotion 11?

Same question would apply for the `jest-emotion` dependency deletion in `packages/calypso-build/package.json`

Reply:
I made this change when working through all the dependency issues raised by yarn. I suspect this can be restored. `jest-emotion` didn't appear to be used by calypso-build, which is why I removed it. (We only use it in `client/`, so it's added to that package.json)

Reply:
Oh, the other reason is that `@types/jest` needs to be at a lower version in one of the packages to meet peer dependency requirements.